### ISSUE TITLE
How to access the getResources() method 

### ISSUE THREAD

**Issue by sagarika-halya** [jbj88817/ExpandableListView-example-android#1]
Hi ,

Thanks for providing the nice Example.I have a doubt in this example.How to access the strings from strings file in expandable list data pump class.Right now you are hardcoded, I want those strings from Strings file.Because Iam using multilanguages in my app.Could you please tell me how to do that?  It's Very urgent requirement for me.

Thanks & Regards,
Sagarika.

**Comment by jbj88817** [comment]
@sagarika-halya Are you retrieving the Strings from strings.xml?

**Comment by sagarika-halya** [comment]
Yes now i solved that problem. Now,I am able to retriving the strings.
